.gitignore

```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```

Comment:
@Raziyehbazargan this is signifying a bad merge/merge conflict - you'll want to remove all sections that include the likes of `<<<<<<<<< HEAD` and/or `=========`